doc/README/README.environment

```diff
@@ -4516,3 +4516,10 @@ Variable:  AFNI_ICORR_UBER_USER
 ------------------------------------
 If set to YES, enable distance measures that are not ready for prime
 time in the instacorr interface.
+
+------------------------------------
+Variable:  AFNI_SKIP_TCSV_SCAN
+------------------------------------
+If set to YES, do not scan for CSV files at afni startup. Same as using option -notcsv 
+
+time in the instacorr interface.
```

Comment:
these two lines appear can be deleted?  perhaps a copy+paste leftover?

Reply:
Yup, cut and paste left over.

Reply:
Just pushed a cleaned version